pkg/printers/internalversion/printers.go

```diff
@@ -3207,6 +3207,9 @@ func (list SortableResourceNames) Less(i, j int) bool {
 }
 
 func isRestartableInitContainer(initContainer *api.Container) bool {
+	if initContainer == nil {
+		return false
+	}
 	if initContainer.RestartPolicy == nil {
 		return false
 	}
```

Comment:
nit
```suggestion
	if initContainer == nil || initContainer.RestartPolicy == nil {
		return false
	}
```

Reply:
if it's ok, I'd like to leave the nil case standalone since it's a weird exception (I'd also consider backporting this since the nil panic was introduced in 1.28, and I want to make the diff as clean as possible)

Reply:
Makes sense :+1: thanks!